hknweb/events/views.py

```diff
@@ -105,3 +108,11 @@ def add_event(request):
             messages.success(request, 'Something went wrong oops')
             return render(request, 'events/add_event.html', {'form': EventForm(None)})
     return render(request, 'events/add_event.html', {'form': EventForm(None)})
+
+@method_decorator(permission_required('events.change_event', login_url='/accounts/login/'), name='dispatch')
```

Comment:
I think `login_url='/accounts/login/'` is actually the default:
https://docs.djangoproject.com/en/2.2/ref/settings/#std:setting-LOGIN_URL

Reply:
But being explicit isn't bad